app/main.py

```diff
@@ -1,3 +1,63 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (Distance, int, float)) -> Distance:
```

Comment:
In this annotation you are waiting other to be a tuple with 3 arguments?

You did correct in `__init__` method. We can use `|` to provide multiple types here. Fix in all cases pls